src/main/java/com/conveyal/datatools/manager/jobs/OtpRunnerManifest.java

```diff
@@ -8,12 +8,11 @@
  */
 public class OtpRunnerManifest {
     public String baseFolder;
-    public List<String> baseFolderDownloads;
+    public List<OtpRunnerBaseFolderDownload> baseFolderDownloads;
     public String buildConfigJSON;
     public boolean buildGraph;
     public String buildLogFile;
-    public String graphObjUrl;
-    public List<String> gtfsDownloads;
+    public String graphObjUri;
     public String jarFile;
     public String jarUrl;
```

Comment:
`jarUrl` should be `jarUri` (also apply to getter/setter).

Reply:
Fixed in https://github.com/ibi-group/datatools-server/pull/363/commits/d30a9edb8db1fe4acf1bc521ecae471938349f05